core/src/main/java/com/linecorp/armeria/common/DefaultRequestHeadersBuilder.java

```diff
@@ -112,4 +118,35 @@ public RequestHeadersBuilder authority(String authority) {
         setters().authority(authority);
         return this;
     }
+
+    @Override
+    public List<LanguageRange> acceptLanguages() {
+        final HttpHeadersBase getters = getters();
+        if (getters != null) {
+            return getters.acceptLanguages();
+        }
+        return DefaultRequestHeaders.ACCEPT_LANGUAGE_WILDCARD;
```

Comment:
nit: `HttpHeadersBase.ACCEPT_LANGUAGE_WILDCARD;`

Reply:
Removed `DefaultRequestHeaders.ACCEPT_LANGUAGE_WILDCARD` completely and returns null.